src/main/java/org/reaktivity/nukleus/maven/plugin/internal/ast/parse/AstParser.java

```diff
@@ -522,6 +551,17 @@ public AstNode visitType_id(
         return super.visitType_id(ctx);
     }
 
+    private void putAbsoluteName(
+        String name)
+    {
+        if (scopeBuilders.peekLast() != null)
+        {
+            String absoluteName = new StringBuilder().append(scopeNameMap.get(scopeBuilders.peekLast().name()))
+                .append("::").append(name).toString();
```

Comment:
Use `String.format(...)` instead.
Use the term `qualifiedName` instead of `absoluteName`.